### ISSUE TITLE
Feature Request: Implement "Page" column

### ISSUE THREAD

**Issue by sakapoodle** [RePod/psdle#83]
Requesting to implement "page" column in the output list.

In addition to the 5 columns currently displayed (Icon, Name, Platform, Size, Date) it would be helpful if "Page" was also displayed.
This would help locate the necessary item faster in the "Download List".

Integer division should easily solve this, something like: `page = 1 + (current_item_number / 24)`,  
where `current_item_number` increments by 1, I believe it's already implemented as either `index` or `id`.

### Example case:
I recently wanted to download "Duke Nukem 3D: Megaton Edition" so I found the URL to queue it:
https://store.playstation.com/en-se/product/EP3643-NPEB01882_00-DUKE3DHDEUPS3000

Unfortunately it seems that it has been removed from the store, so my only option is to locate it in the Download List.
Currently I have 2219 items across 93 pages of the Download List, so it's impractical to search for the item manually.

To narrow down the search:
```
Sakapoodle> grep -in "megaton" account.csv

1008:Megaton Rainfall,,PS4,https://store.playstation.com/en-se/product/EP1055-CUSA06563_00-MEGATONRAINFALL0,
1552:Duke Nukem 3D: Megaton Edition,Duke Nukem 3D: Megaton Edition,PS Vita,https://store.playstation.com/en-se/product/EP3643-NPEB01882_00-DUKE3DHDEUPS3000,
1553:Duke Nukem 3D: Megaton Edition,Duke Nukem 3D: Megaton Edition,PS3,https://store.playstation.com/en-se/product/EP3643-NPEB01882_00-DUKE3DHDEUPS3000,
```

Ok, it's on line 1553 of the CSV file, but how many lines are there in total?

```
Sakapoodle> wc -l account.csv
2307 account.csv
```

2307 lines.. ok so some basic arithmetics:
1553/2307 = 0.673

So about 67.3% into the CSV file is where the game is located, which means it should also be about 67.3% into the 93 pages of the Download List.

0.673 * 93 ≈ 62.5 pages in.
Around page 62 or 63 is where it's probably at..

![psdle-found-target](https://user-images.githubusercontent.com/43319787/89915681-f3fb6c80-dbf6-11ea-8026-8ff20d1cab2d.png)

NOTE: There's some discrepancy because PSDLE reports 2307 items on my account while Download List shows 2219 items.

With a "page" column for the item I would not have to calculate this myself. :smiley:

**Comment by RePod** [comment]
I had originally typed that this already exists, albeit not in property form, when hovering over game icons on the list but this report exposed that at some point they changed the page size from 50 to 24 items. This has been updated. I'll assume since they brought the number down it has parity with mobile. I never went looking for where the value to paginate at is stored so let's hope it doesn't change again.

I'll slip `page` in as a regular property using the [existing math](https://github.com/RePod/psdle/blob/5367c78a09f98c6fb8f395325191867a40b42724/_src/psdle.base.js#L1586 "pls dont laugh") `Math.ceil(val.index/pg)` and it should work no problem.

As for the discrepancy in item count it eliminates video, service, and expired content then anything left with entitlement data it parses. My discrepancy is 9 items which seem to be these `N/A` sized PS4 items from a "set":
![](https://user-images.githubusercontent.com/7897871/89933086-893d3780-dbdc-11ea-8e19-9502f6a08f81.png)
The "Yusuke" set doesn't appear on my list at all, while the "Morgana" and "Ryuji" sets do but only as one item. The only other `N/A` item is Driveclub PS+ Edition which *does* show up. If you can verify any similar characteristics of bundles/sets I'll look into it.

**Comment by RePod** [comment]
Now available in 3.3.18 as `dlListPage` with a few other new properties.

**Comment by RePod** [comment]
Pages are no more.

This may be relevant in the future when lists have more than 1000 items again.
